index.html

```diff
@@ -2455,113 +2455,129 @@ <h2>Services</h2>
     <h1>Representations</h1>
 
     <p>
-All concrete <a>representations</a> of a <a>DID document</a> are serialized using a
-deterministic mapping that is able to be unambiguously parsed into the <a
-href="#data-model">data model</a> defined in this specification. All
-serialization methods MUST define rules for the bidirectional translation of a
-<a>DID document</a> both into and out of the <a>representation</a> in question. An
-implementation MUST NOT convert between <a>representations</a> without first parsing to
-a <a href="#data-model">data model</a> (described in Sections <a
-href="#data-model"></a> and <a href="#core-properties"></a>); translation
-between any two <a>representations</a> is done by parsing the source <a>representation</a>
-into the <a href="#data-model">data model</a> and then serializing the <a
-href="#data-model">data model</a> into the target <a>representation</a>.
+A concrete serialization of any data structure in this specification is called a
```

Comment:
```suggestion
A concrete serialization of a <a>DID document</a> in this specification is called a
```

Let's not change this please, especially since the text right after this sentence refers to the DID document data model. I imagine you're probably trying to cover the metadata structures as well, but I think it's better to keep those separate from the DID document data model and representations for now...

Reply:
We need to change this in order to make the DID Resolution section testable. There has to be a clear path to serialization for the data structures in the Resolution section, namely the metadata structure. We can achieve that by making sure the ADM isn't hard coded to just apply to DID Documents. I have made the changes in each section and it all seems to stick together fairly well. We can discuss on the call today. If we don't make this change, we're back to the Resolution section not having a clear path to a testable serialization.

Reply:
I don't think this proposed change does what you think it does @peacekeeper ... we define a DID Document as an INFRA map.... I would be happy to take the change.... Since a DID Document is just an INFRA map, with a required property of `id`.

Reply:
Applied @peacekeeper's change per DID WG call resolution: https://www.w3.org/2021/02/16-did-topic-minutes.html#r01